sdk/react/upNext.js

```diff
@@ -83,7 +83,7 @@ var UpNext = React.createClass({
   },
 
   _renderDismissButton: function() {
```

Comment:
Where the docs at?!?!?!

Reply:
fair enough, but to be honest I think we need to work on docs on all the files in our JS code since we have none. Should I start on this one or do I wait and start on another PR just dedicated to docs?

Reply:
To minimize downtime, if you touch a function, look at it's jsdocs.  You don't have to do the whole file.  We'll chip away at it slowly.